extensions/data-transfer/portability-data-transfer-apple/src/main/java/org/datatransferproject/datatransfer/apple/photos/AppleMediaInterface.java

```diff
@@ -443,15 +444,15 @@ public int importAlbums(
             mediaAlbum.getName(),
             () -> {
               monitor.severe(
-                () -> "Error importing album: ",
+                () -> "Fail to create album: ",
                 AuditKeys.jobId, jobId,
                 AuditKeys.albumId, mediaAlbum.getId(),
                 AuditKeys.errorCode, newPhotoAlbumResponse.getStatus().getCode());
 
-              throw new IOException(
+              throw new IOException(getApplePhotosImportThrowingMessage(
                 String.format(
-                  "Failed to create album, error code: %d",
-                  newPhotoAlbumResponse.getStatus().getCode()));
+                  "Fail to create album, error code: %d",
+                  newPhotoAlbumResponse.getStatus().getCode())));
```

Comment:
Logging & throwing is unnecessary, we should usually only do one of the two. 

If we log here, the client which received the error may double-log, or they may throw the error leaving the logs in the stack trace